### DISCUSSION TITLE
How to Create Subdirectory

### DISCUSSION THREAD

**Discussion by 0maunz0** [redimp/otterwiki#168] [Q&A] [answered]
What is the correct way to create a subdirectory?
Currently, I create a page which is going to be my subdirectory.
Then I create my page with the name subdirectory/newpage.
 
However, this creates a folder as well as a page - which means that the page index is not displayed when i navigate to the subdirectory.
I checked the files and there is a subdirectory.md as well as the folder present.
Deleting the subdirectory.md solves the problem an the page index works as expected.

Am I doing something wrong when creating the subdirectory, or is there an issue?

**Comment by redimp** [comment, marked as answer]
Hey @0maunz0, Subdirectories are currently only created while creating a page, like you said, <kbd>Create Page</kbd> `Subdir/Example` <kbd>Save</kbd>. This created `Subdir/Example.md`, but not `Subdir.md`. Can you please double check, that you not created `Subdir` in addition to `SubDir/Example`?

**Reply by 0maunz0** [reply]
Thank you for the quick reply! I did it wrong then, since i tried to first create a subfolder alone, without the page - which creates just a page. Creating the page with the subdir/example works as expected.
Also a huge thank you for this project, really hits the mark in terms of being simple and yet with enough features to be useful.
